index.html

```diff
@@ -1,28 +1,97 @@
-    </div>
+        <ul class="tags">
+          <li class="tags__item">first</li>
+          <li class="tags__item">second</li>
+          <li class="tags__item">third</li>
+          <li class="tags__item">third</li>
+          <li class="tags__item">third</li>
+        </ul>
+      </section>
+      <section class="card">
+        <button class="card__bookmark"></button>
+        <h2 class="card__question">Frage</h2>
+        <button class="card__button">Antwort anzeigen</button>
+        <p class="card__answer card__answer--hidden" hidden>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Nobis laborum
+          itaque consequuntur deserunt inventore amet repellat labore quaerat
+          est facere natus, explicabo eum velit quidem.
+        </p>
+        <ul class="tags">
+          <li class="tags__item">first</li>
+          <li class="tags__item">second</li>
+          <li class="tags__item">third</li>
+          <li class="tags__item">third</li>
+          <li class="tags__item">third</li>
+        </ul>
+      </section>
+      <section class="card">
+        <button class="card__bookmark"></button>
+        <h2 class="card__question">Frage</h2>
+        <button class="card__button">Antwort anzeigen</button>
+        <p class="card__answer card__answer--hidden" hidden>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Nobis laborum
+          itaque consequuntur deserunt inventore amet repellat labore quaerat
+          est facere natus, explicabo eum velit quidem.
+        </p>
+        <ul class="tags">
+          <li class="tags__item">first</li>
+          <li class="tags__item">second</li>
+          <li class="tags__item">third</li>
+          <li class="tags__item">third</li>
+          <li class="tags__item">third</li>
+        </ul>
+      </section>
+      <section class="card">
+        <button class="card__bookmark"></button>
+        <h2 class="card__question">Frage</h2>
+        <button class="card__button">Antwort anzeigen</button>
+        <p class="card__answer card__answer--hidden" hidden>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Nobis laborum
+          itaque consequuntur deserunt inventore amet repellat labore quaerat
+          est facere natus, explicabo eum velit quidem.
+        </p>
+        <ul class="tags">
+          <li class="tags__item">first</li>
+          <li class="tags__item">second</li>
+          <li class="tags__item">third</li>
+          <li class="tags__item">third</li>
+          <li class="tags__item">third</li>
+        </ul>
+      </section>
+    </main>
+    <footer>
+      <nav class="footer__nav">
+        <a href="#"><img src="/img/Home.svg" width="70" alt="" /></a>
+        <a href="#"><img src="/img/bookmark.svg" width="45" alt="" /></a>
+        <a href="#"><img src="/img/book-plus.svg" width="70" alt="" /></a>
+        <a href="#"
+          ><img src="/img/human-male-board.svg" width="70" alt=""
+        /></a>
```

Comment:
Same as the comment above, it would really be good if you set the `alt` attribute here or consider adding text as well, so that the navbar is clearer!